src/clients/ntp-client_proposal.rb

```diff
@@ -124,7 +143,14 @@ def ui_enable_disable_widgets(enabled)
         else
           UI.ChangeWidget(Id(:ntp_now), :Enabled, enabled)
         end
+
         UI.ChangeWidget(Id(:ntp_save), :Enabled, enabled)
+        UI.ChangeWidget(Id(:ntp_address), :Enabled, enabled)
+
+        @sources_table.send(enabled ? :enable : :disable)
```

Comment:
this piece of code is used from ```yast2-country``` to drive availability of the ntp piece of dialog. Basically, if radio button ```Synchronize with NTP Server``` is not checked, everything in the section should be disabled. That is driven by method which receives ``` enabled = true|false```.  ```send``` is used because it looked more pretty to me then ```if enabled then enable else disable end``` (or similar) approach.

And why ```enable|disable``` instead of ```enabled?``` ? Well AFAIK first changes the state of widged, the second just reports it.

Reply:
> this piece of code is used from `yast2-country` to drive availability of the ntp piece of dialog. Basically, if radio button `Synchronize with NTP Server` is not checked, everything in the section should be disabled. That is driven by method which receives ` enabled = true|false`. `send` is used because it looked more pretty to me then `if enabled then enable else disable end` (or similar) approach.

Understood, but then it is probably to use **public_send** instead..
> 
> And why `enable|disable` instead of `enabled?` ? Well AFAIK first changes the state of widged, the second just reports it.

true